src/core/cgroup.h

```diff
@@ -104,7 +104,8 @@ struct CGroupBPFForeignProgram {
 
 struct CGroupSocketBindItem {
         LIST_FIELDS(CGroupSocketBindItem, socket_bind_items);
-        int address_family;
+        int32_t address_family;
+        int32_t ip_protocol;
```

Comment:
ah, uh. libc uses "int" for both these things, see socket() prototype, hence I actually think we should use `int` to wherever we can. However there's one place where we shouldn't: when passing a pointer to these field to sd_bus_message_read(), because theoretically int and int32_t might be two different things...

hence, please continue to use `int` here. However, when invoking sd_bus_message_read() pass pointers to true int32_t variables, and then assign from there to the `int` fields the value sactually should end up in.

i.e. the mess around in32_t vs. int should remain specific to sd_bus_message_read()/D-Bus, which causes it. Once we got our data out of that, let's not leak the type confusion and let's use libc's preferred type for this everywhere else, i.e. `int`, if you follow what i mean